packages/vjudge/src/model.ts

```diff
@@ -55,6 +55,11 @@ class Service {
 
     async sync(domainId: string, resync = false, list: string) {
         let page = 1;
+        if (domainId !== 'codeforces') {
+            const oj = await coll.findOne({ type: domainId });
+            page = oj.page;
```

Comment:
L57 show that Hydro default sync from first page. You should deal with the first sync without page.